Add debug buffer support for pybindings

lucylq opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #13987
* #14024

Add debug buffer to PyModule. Pass this to the event tracer and collect it back later.

Differential Revision: [D81751067](https://our.internmc.facebook.com/intern/diff/D81751067/)

lucylq committed:
Add debug buffer support for pybindings

Add debug buffer to PyModule. Pass this to the event tracer and collect it back later.

Differential Revision: [D81751067](https://our.internmc.facebook.com/intern/diff/D81751067/)

[ghstack-poisoned]

kimishpatel reviewed (approved):
Review automatically exported from Phabricator review in Meta.

lucylq committed:
Update on "Add debug buffer support for pybindings"


Add debug buffer to PyModule. Pass this to the event tracer and collect it back later.

Differential Revision: [D81751067](https://our.internmc.facebook.com/intern/diff/D81751067/)

[ghstack-poisoned]

lucylq committed:
Update on "Add debug buffer support for pybindings"


Add debug buffer to PyModule. Pass this to the event tracer and collect it back later.

Differential Revision: [D81751067](https://our.internmc.facebook.com/intern/diff/D81751067/)

[ghstack-poisoned]
